src/@types/express-session/index.d.ts

```diff
@@ -17,5 +17,7 @@ declare module 'express-session' {
     dataset: {
       [id: string]: DatasetProps;
     };
+    buildNextAction: string | undefined;
+    buildPreviousAction: string | undefined;
```

Comment:
These should be namespaced under the `dataset` key above by placing them in `DatasetProps`, otherwise different datasets can interfere (especially in tests running in parallel for the same user).